### PR TITLE
Update integrate.md

### PR DESCRIPTION
**Pull request by GMerakis**
Include an explanation on why the code won't compile

*Issue #, if available:*
#3035

*Description of changes:*
There are multiple questions regarding this issue, few solutions.
The issue can actually be solved by following the tutorial but this was not called out in the doc

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/start/getting-started/fragments/flutter/integrate.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 In this tutorial, you will integrate basic functionality for **Analytics**.
 
+**Note**: the code won't compile until after you have finished <b>*'Connect to the cloud'*</b> section of the tutorial.
```
Comment (jamesonwilliams) [inline docs/start/getting-started/fragments/flutter/integrate.md:3]: `<amplify-callout>` perhaps?

**Review by jamesonwilliams — APPROVED** [review]
`<amplify-callout>` perhaps?

**Comment by GMerakis** [comment]
> `<amplify-callout>` perhaps?

done

**Comment by palpatim** [comment]
@haverchuck Can you or one of the Flutter team take a look at this & merge it in if it meets with your approval?

File: docs/start/getting-started/fragments/flutter/integrate.md [hunk 2]
```diff
@@ -1,5 +1,7 @@
 In this tutorial, you will integrate basic functionality for **Analytics**.
 
+<amplify-callout> The code won't compile until after you have finished <b>*'Connect to the cloud'*</b> section of the tutorial. </amplify-callout>
```
Comment (drochetti) [inline docs/start/getting-started/fragments/flutter/integrate.md:3]: in order to render markdown correctly, `amplify-callout` requires you to skip a line after the opening tag and before closing it. Also, plain markdown is preferred over `<b>`.

That being said, I think a better fit for it is to actually link to the section of the tutorial you mentioned, here's my suggestion.

```suggestion
<amplify-callout>

The code won't compile until after you have finished [Connect to the cloud](~/start/getting-started/integrate.md) section of the tutorial.

</amplify-callout> 
```

**Review by drochetti — CHANGES_REQUESTED** [review]

**Pull request closed without merging**